server/game/costs/payxgoldprompt.js

```diff
@@ -18,21 +19,29 @@ class PayXGoldPrompt extends BaseStep {
 
         let range = _.range(this.min, this.max + 1).reverse();
 
-        let buttons = _.map(range, gold => {
-            return { text: gold.toString(), method: 'resolveCost', arg: gold };
+        let buttons = _.map(range, xValue => {
+            return { text: xValue.toString(), method: 'resolveCost', arg: xValue };
         });
 
+        let titleStr;
+        if(this.reduction > 0) {
+            titleStr = 'Select value of X (before reduction of ' + this.reduction + ')';
```

Comment:
I'm not sure we need this specific of a title. It's also a bit long. I think just offering the buttons for the reducer-boosted gold values with the old title is sufficient.